soccer/radio/PacketConvert.cpp

```diff
@@ -72,6 +73,59 @@ Packet::RadioRx convert_rx_rtp_to_proto(const rtp::RobotStatusMessage& msg) {
     return packet;
 }
 
+void construct_tx_proto(
+    Packet::RadioTx& radioTx,
```

Comment:
You know, on second thought I'd actually rather we take out all of the protobuf stuff from the radio interface and do the log struct filling in Processor.